geostore/import_metadata_file/task.py

```diff
@@ -16,6 +16,7 @@
 from ..types import JsonObject
 
 S3_BODY_KEY = "Body"
+CATALOG_FILENAME = "catalog.json"
```

Comment:
This constant already exists in `geostore/populate_catalog/task.py`, so it should probably be moved to a file in the `geostore` directory and imported.

Reply:
Have changed this to reference `geostore/populate_catalog/task.py` (for now). Good idea to move this to another file, but a quick look shows it is reference in quite a few places, so I might move this change to another PR.

Reply:
Spoke too soon. The value needs to be accessible via the lambda function. Changing this back to `../catalog.json` for now and will create another PR to address the above.